CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
Alright, let me discuss it some more internally and we'll get back on the search box stuff.